ui/lib/objects.ts

```diff
@@ -1,13 +1,13 @@
 import { stringify } from "yaml";
-import { addKind } from "./utils";
 import {
-  GitRepositoryRef,
   Condition,
   FluxObjectKind,
   FluxObjectRef,
-  Object as ResponseObject,
+  GitRepositoryRef,
   Interval,
+  Object as ResponseObject,
 } from "./api/core/types.pb";
+import { addKind } from "./utils";
```

Comment:
Changes in `objects ts` are reported by github a bit weirdly, I hope it will clear out after rebase.

Reply:
Looks like a linting thing - is my formatting on save messed up? Or has this file just not gone through it yet?